frontend/vue/components/CodeExercise/CodeOutput.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div ref="outputDiv" class="code-output">
+    <div class="code-output__state-info__wrapper">
+      <div v-if="!kernel && !error" class="code-output__state-info">
+        <bx-loading class="code-output__state-info__icon" assistive-text="Connecting to the server" type="small" />
+        Connecting to the server
+      </div>
+      <div v-if="error !== ''" class="code-output__state-info">
+        Failed to execute. {{ error }} Please refresh the page.
```

Comment:
Similar comment regarding translations, but not sure what the syntax would be to include the `{{ error }}` message.